docs/src/Buttons/Stories.elm

```diff
@@ -104,49 +100,74 @@ Button.bodyIcon Icon.someIcon
 primaryStory cfg =
     enabledStory cfg
         "Primary"
-        Button.tonePrimary
-        "Button.tonePrimary"
+        Button.primary
+        "Button.primary"
 
 
 dangerStory cfg =
     enabledStory cfg
         "Danger"
-        Button.toneDanger
-        "Button.toneDanger"
+        Button.danger
+        "Button.danger"
 
 
-successStory cfg =
-    enabledStory cfg
-        "Success"
-        Button.toneSuccess
-        "Button.toneSuccess"
+successStory renderConfig =
+    storyList
+        ( "Success"
+        , [ iconsSvgSprite
+          , Button.fromLabel "Prompt"
+                |> Button.success
+                |> Button.renderElement renderConfig
+          , Button.fromIcon (Icon.toggle "Toggle")
+                |> Button.cmd Msg.NoOp Button.primary
+                |> Button.withSuccessIf True
+                |> Button.renderElement renderConfig
+          ]
+        , { note = """
+```elm
+-- Text
+
+Button.fromLabel "Some Text"
+    |> Button.success
+    |> Button.renderElement renderConfig
+
+
+-- Icon
+Button.fromIcon Icon.someIcon
+    |> Button.cmd YourMessage
+    |> Button.withSuccessIf True
+    |> Button.renderElement renderConfig
+```
+"""
+          }
+        )
 
 
 lightStory cfg =
     enabledStory cfg
         "Light"
-        Button.toneLight
-        "Button.toneLight"
+        Button.light
+        "Button.light"
 
 
 clearStory cfg =
     enabledStory cfg
         "Clear"
-        Button.toneClear
-        "Button.toneClear"
+        Button.clear
+        "Button.clear"
 
 
 linkStory renderConfig =
     story
         ( "Link"
-        , Button.bodyText "Go to Blank"
-            |> Button.link (Link.link "about:blank")
-            |> Button.toEl renderConfig
+        , Button.fromLabel "Go to Blank"
+            |> Button.redirect (Link.link "about:blank") Button.hyperlink
```

Comment:
Yeah, I think we don't need Link module anymore, this line could be only: 

`|> Button.redirect "about:blank" Button.hyperlink`

and then we move everything is needed from UI.Link to this module as internal functions

Reply:
if is that the case, we can move to an Internal module and still do something like:

`|> Button.redirect "about:blank" Button.hyperlink`

I'm not sure where the user would use this link, if only the components are using, we can set as an internal thing